scripts/create_test

```diff
@@ -327,12 +336,6 @@ def create_test(testargs, compiler, machine_name, no_run, no_build, no_batch, te
                 else:
                     expect(machine_name == testsplit[4],
                            "ambiguity in machine, please use the --machine option")
-            if testsplit[5] is not None:
```

Comment:
I fixed it for xml tests but not for tests listed on the command line or in a testfile.